sdk/cosmosdb/cosmos/samples/v4/javascript/ContainerManagement.js

```diff
@@ -157,8 +157,19 @@ async function run() {
     spatialIndexes: [{ path: "/location/*", types: ["Point", "Polygon"] }],
     vectorIndexes: [
       { path: "/vector1", type: VectorIndexType.Flat },
-      { path: "/vector2", type: VectorIndexType.QuantizedFlat },
-      { path: "/vector3", type: VectorIndexType.DiskANN },
+      {
```

Comment:
@ujjwalsoni1707 this is an autogenerated right?

Reply:
Yes changes were done in dev-samples and then samples were autogenerated.